apps/vr-tests/.storybook/main.js

```diff
@@ -1,6 +1,37 @@
-const { createStorybookWebpackConfig } = require('@fluentui/scripts-webpack');
+// @ts-check
 
+const path = require('path');
+const { registerRules, registerTsPaths, rules } = require('@fluentui/scripts-storybook');
+
+const tsConfigPath = path.resolve(__dirname, '../../../tsconfig.base.v8.json');
 module.exports = /** @type {import('../../../.storybook/main').StorybookBaseConfig} */ ({
+  addons: [
+    {
+      name: 'storybook-addon-swc',
+      options: /** @type {import('storybook-addon-swc').StoryBookAddonSwcOptions} */ ({
+        swcLoaderOptions: {
```

Comment:
are the swc configs duplicated between the two vr test apps?

Reply:
they are, I'll consider encapsulation as follow-up. but it should be handled by generator to be up to date/in sync